src/models/searchservice/response/HighlightedFields.ts

```diff
@@ -0,0 +1,58 @@
+import { HighlightedValue } from './HighlightedValue';
+
+/**
+ * A mapping of the values emphasized by the Answers API.
```

Comment:
nit: What do you think about updating this to "A mapping of fields to the values emphasized by the Answers API"?

Reply:
yeah I like that